CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   This reduces the API surface of the crate which allows for fewer breaking changes in the future.
   All functionality from `Docker` (start, stop, rm, and ports) is available on a container directly.
 - `descriptor` is broken down into `name` and `tag` within `Image` trait.
+- [shiplift](https://github.com/softprops/shiplift) replaced with [bollard](https://github.com/fussybeaver/bollard).
```

Comment:
Okay, so this is a bit embarrassing. I've been holding off to make a release for so long that we are replacing the initial implementation of this with something else and the initial one wasn't even release yet!

As such, I think we don't need to mention this because for people upgrading from `0.12`, they didn't even know that this was even based on `shiplift` ...

Reply:
Ha! :) Yeah, I know a thing or two about delaying a release :) Removed.